hack/build.sh

```diff
@@ -110,30 +115,19 @@ go_fmt() {
   find $(echo $source_dirs) -name "*.go" -print0 | xargs -0 gofmt -s -w
 }
 
-# Run a go tool, get it first if necessary.
-run_go_tool() {
```

Comment:
This function has been removed but is still referenced below ?

Reply:
I've opted for the common one from hack repo, https://github.com/knative/hack/blob/main/library.sh#L489-L514.